src/components/chat/Message.js

```diff
@@ -7,6 +9,9 @@ class Message extends Component {
     let { message, username } = this.props;
     let fromMe = message.from === username;
     let messageClass = fromMe ? 'chat-outgoing' : 'chat-incoming';
+    
+    let RFCDate = message.key.split('-')[0];
+    let date = formatRelative(Date.parse('Fri, 01 Sep 2016 03:42:50 GMT'), new Date());
```

Comment:
@iamrobinpham Was this supposed to be `formatRelative(Date.parse(RFCDate), new Date())`, or perhaps something else using `RFCDate`?

Reply:
oops yeah! you got it `formatRelative(Date.parse(RFCDate), new Date())` is correct. I must've committed test code :(.

Reply:
Okay, feel free to edit, commit the fix, and do a forced push to update this branch :-)